core/algorithms/onpolicy_sync/runner.py

```diff
@@ -448,16 +448,12 @@ def save_project_state(self):
 
         # Recursively saving configs
         if self.loaded_config_src_files is not None:
-            for file in self.loaded_config_src_files:
-                base, module = self.loaded_config_src_files[file]
-                parts = module.split(".")
-
-                src_file = os.path.sep.join([base] + parts) + ".py"
+            for src_file in self.loaded_config_src_files:
                 assert os.path.isfile(src_file), "Config file {} not found".format(
                     src_file
                 )
 
-                dst_file = os.path.join(base_dir, os.path.join(*parts[1:]),) + ".py"
+                dst_file = os.path.join(base_dir, os.path.basename(src_file))
```

Comment:
what if two files have the same name in different folders?

Reply:
I think we should save the entire relative path

Reply:
I realized that even using relative paths is not necessarily sufficient as weird things can happen if you run `main.py` outside of the top-level directory. For now I've added some code to avoid name conflicts by adding a prefix if necessary.